### PR TITLE
Minify script and module

### PR DESCRIPTION
**Pull request by naveed-fida**
Sorry @aamirafridi, I dropped the ball on this one. Got tangled up in some other work. Saw your commit to the readme file and had to start working on this. Can you take a quick look? If not I can merge directly.

I've added support for npm modules. So now users can do the following:

```
const $ = require("jquery");
require("jquery.marquee");
```

Tested this on node.js using a browserify build. Works and hasn't broken the other usage scenario.

Also added an npm script for minifying the js file.

### CODE REVIEW

**Comment by naveed-fida** [comment]
@aamirafridi, from now on, I'll keep an eye on this repo and maintain it as well as I can.

File: .gitignore
Comment (AamirAfridi-Which) [inline .gitignore:3]: Remove this file from gitignore. We should commit this file
Can you also add .nvmrc and .npmrc files as well?

File: package.json
Comment (AamirAfridi-Which) [inline package.json:7]: nice

File: jquery.marquee.js
Comment (AamirAfridi-Which) [inline jquery.marquee.js:4]: Can you please updated ReadMe file to add instructions about how to use in node project
Does it work with import as well?

Can also update ReadMe to give example of how to use this plugin in a React component. Something like this https://reactjs.org/docs/integrating-with-other-libraries.html#how-to-approach-the-problem but useEffect hook
If it's too much then leave it :)

**Review by AamirAfridi-Which — CHANGES_REQUESTED** [review]
Hi @naveed-fida really good work. Thanks for that
Some comments when you have time
After merging this we should publish this and also update https://www.jsdelivr.com/package/npm/jquery.marquee

File: package.json
Comment (naveed-fida) [reply package.json:7]: Thanks

File: .gitignore
Comment (naveed-fida) [reply .gitignore:3]: Yep. Added that by mistake.

File: jquery.marquee.js
Comment (naveed-fida) [reply jquery.marquee.js:4]: Will do. Shouldn't be too much.

Comment (naveed-fida) [reply jquery.marquee.js:4]: Should I remove the "looking for maintainer" text from the top of the readme?

Comment (AamirAfridi-Which) [reply jquery.marquee.js:4]: That's up to you ;)
It will be nice if you do. You will be supporting a lib that has almost 100 million downloads :)
If you do this, can you please review all the issues and address/close them?
Also, keep an eye on pull requests

I will give you access to npm publish and jsdelivr so you can publish them yourself
I am also thinking that expose a React component (transpiled file) where you can pass all options as props

**Comment by AamirAfridi-Which** [comment]
@naveed-fida please ping me once the comments are addressed and I can review them again 
Thanks for looking after this orphan plugin :)

File: jquery.marquee.js
Comment (naveed-fida) [reply jquery.marquee.js:4]: Sure. I'll remove the maintainer announcement. Re the react transpiled component, I'll think about it. Doesn't seem highly useful to me as building a custom component with the plugin shouldn't be difficult, and it adds new dependencies to our project. Still, could be useful as a quick solution.

Comment (AamirAfridi-Which) [reply jquery.marquee.js:4]: if you add all the dependencies as peer-dependencies then it should be fine
But that's for another day, for now just a basic React example in ReadMe is enough

**Comment by naveed-fida** [comment]
Over to you @AamirAfridi-Which

File: README.md
Comment (AamirAfridi-Which) [inline README.md:142]: Absolutely amazing :)

**Review by AamirAfridi-Which — APPROVED** [review]

**Review by AamirAfridi-Which — APPROVED** [review]

**Comment by aamirafridi** [comment]
@naveed-fida please send me your email address and npm user name
I am trying to give you full access on jsdelivr and npm so you can publish easily

**Comment by naveed-fida** [comment]
@aamirafridi check your messenger for my email and npm username

**Pull request merged**
